filter.imp

```diff
@@ -1,13 +1,111 @@
+   # bit twiddling operations should use linux/bits.h
+   { include: ["@.asm-generic/bitsperlong.h.", private, "<linux/bits.h>", "public"] },
+   { include: ["@.asm-generic/bpf_perf_event.h.", private, "<asm/bpf_perf_event.h>", "public"] },
+   { include: ["@.asm-generic/byteorder.h.", private, "<asm/byteorder.h>", "public"] },
    { include: ["@.asm-generic/errno.h.", private, "<linux/errno.h>", "public"] },
-   { include: ["@.asm-generic/errno-base.h.", private, "<linux/errno.h>", "public"] },
    { include: ["@.asm/errno.h.", private, "<linux/errno.h>", "public"] },
-   { include: ["@.asm/bug.h.", private, "<linux/bug.h>", "public"] },
+   { include: ["@.asm-generic/fcntl.h.", private, "<asm/fcntl.h>", "public"] },
+   { include: ["@.asm-generic/ioctl.h.", private, "<asm/ioctl.h>", "public"] },
+   { include: ["@.asm-generic/ioctls.h.", private, "<asm/ioctls.h>", "public"] },
+   { include: ["@.asm-generic/ipcbuf.h.", private, "<asm/ipcbuf.h>", "public"] },
+   { include: ["@.asm-generic/mman.h.", private, "<asm/mman.h>", "public"] },
+   { include: ["@.asm-generic/msgbuf.h.", private, "<asm/msgbuf.h>", "public"] },
+   { include: ["@.asm-generic/param.h.", private, "<asm/param.h>", "public"] },
+   { include: ["@.asm-generic/poll.h.", private, "<asm/poll.h>", "public"] },
+   { include: ["@.asm-generic/posix_types.h.", private, "<asm/posix_types.h>", "public"] },
+   { include: ["@.asm-generic/ptrace.h.", private, "<asm/ptrace.h>", "public"] },
+   { include: ["@.asm-generic/resource.h.", private, "<linux/resource.h>", "public"] },
+   { include: ["@.asm/resource.h.", private, "<linux/resource.h>", "public"] },
+   { include: ["@.asm-generic/sembuf.h.", private, "<asm/sembuf.h>", "public"] },
+   { include: ["@.asm-generic/setup.h.", private, "<asm/setup.h>", "public"] },
+   { include: ["@.asm-generic/shmbuf.h.", private, "<asm/shmbuf.h>", "public"] },
+   { include: ["@.asm-generic/sigcontext.h.", private, "<asm/sigcontext.h>", "public"] },
+   { include: ["@.asm-generic/siginfo.h.", private, "<asm/siginfo.h>", "public"] },
+   { include: ["@.asm(-generic)?/signal(-defs)?.h.", private, "<linux/signal.h>", "public"] },
+   { include: ["@.asm-generic/socket.h.", private, "<asm/socket.h>", "public"] },
+   { include: ["@.asm-generic/sockios.h.", private, "<asm/sockios.h>", "public"] },
+   { include: ["@.asm-generic/stat.h.", private, "<asm/stat.h>", "public"] },
+   { include: ["@.asm-generic/statfs.h.", private, "<asm/statfs.h>", "public"] },
+   { include: ["@.asm-generic/swab.h.", private, "<asm/swab.h>", "public"] },
+   { include: ["@.asm-generic/termbits.h.", private, "<asm/termbits.h>", "public"] },
+   { include: ["@.asm-generic/termios.h.", private, "<asm/termios.h>", "public"] },
+   { include: ["@.asm-generic/types.h.", private, "<asm/types.h>", "public"] },
+   { include: ["@.asm-generic/unistd.h.", private, "<asm/unistd.h>", "public"] },
+
+   # This list was generated from include/asm-generic/Kbuild
```

Comment:
However you update the comment from the above block, you should mirror that comment exactly here, modulo the different file path.